tests/data/protocols/protobuf/map_scalar_keys.proto

```diff
@@ -0,0 +1,21 @@
+// map_scalar_keys.proto
+
+syntax = "proto3";
+
+message MapScalarKeys {
+  // map<double, int32> double_key = 1;
+  // map<float, int32> float_key = 2;
```

Comment:
`double`, `float` and `bytes` are not valid key types in protobuf. Left them outcommented here just for sake of completeness; show they have not been forgotten